weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -225,10 +228,9 @@ def _write_dataset(self, dataset: xr.Dataset, split_dims: t.List[str]) -> None:
         # returning bytes. Further, the scipy engine does not support NETCDF4 (which is HDF5 compliant).
         # Storing data in HDF5 is advantageous since it allows opening NetCDF files with buffered readers.
         with tempfile.NamedTemporaryFile() as tmp:
-            with FileSystems().create(self._get_output_for_dataset(dataset, split_dims)) as dest_file:
-                dataset.to_netcdf(path=tmp.name, engine='netcdf4', format='NETCDF4')
-                tmp.seek(0)
-                shutil.copyfileobj(tmp, dest_file, WRITE_CHUNK_SIZE)
+            dataset.to_netcdf(path=tmp.name, engine='netcdf4', format='NETCDF4')
+            tmp.seek(0)
```

Comment:
Why seek 0 here? Does this file object not have a flush method or somilar?

Reply:
Good catch! This is left over from the previous implementation.